packages/bits/src/lib/select-v2/constants.ts

```diff
@@ -4,3 +4,5 @@ import { IOptionedComponent } from "./types";
 
 // Will be renamed in scope of the NUI-5797
 export const NUI_SELECT_V2_OPTION_PARENT_COMPONENT = new InjectionToken<IOptionedComponent>("NUI_SELECT_V2_OPTION_PARENT_COMPONENT");
+export const ANNOUNCER_OPEN_MESSAGE = (optionsLength: number) => `${optionsLength} options available`;
```

Comment:
It would be better to have this as just a vanilla constant. We don't normally use the constant naming convention (`CAPS_WITH_UNDERSCORES`) for functions in Nova. How about the following?

```suggestion
export const ANNOUNCER_OPEN_MESSAGE_SUFFIX = "options available";
```

Then you can use the template literal syntax directly where ever this suffix is needed.

Reply:
resolved it, thanks